tools/abel/README.md

```diff
@@ -0,0 +1,75 @@
+# ABEL
+#### Automated Building Equipment Loader
+
+## Overview
+ABEL is a tool allowing systems integrators to easily and efficiently create [Building Configuration files](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md) and [UDMI Site Models](https://github.com/faucetsdn/udmi/blob/master/docs/specs/site_model.md).
+
+Previously, systems integrators had to a create a building configuration file by hand, but since building config is designed to be a machine=readable document, it is not human-friendly. The ABEL spreadsheet provides a frontend interface for gathering the informtion required to create a Building Config or a Site Model. ABEL also allows users to edit a Building Config file or UDMI Site Model by loading  either document into the frontend spreasheet and exporting it back out.
+
+#### UDMI
+ABEL does not yet support UDMI Site Model, but is planned to in the coming months.
+
+## Setup and installation
+Similar to the [GUID Generator](https://github.com/google/digitalbuildings/tree/master/tools/guid_generator), ABEL's setup process depends on the instance validator. It is best to run the global setup script to setup the entire tools library which ABEL depends on
+
+1. Run: `git clone https://github.com/google/digitalbuildings.git`
+2. Navigate to `DigitalBuildings/tools`
+3. Depending on your operating system run either of the global setup scripts to configure dependencies
+    * Windows: Run `pip_install.bat`
+    * Unix: Run `sudo bash pip_install.sh`
+5. Create a directory for ABEL to use at `~/abel/` and navigate to it
+    * This will be used for storing authentication credentials and documents output from ABEL.
+4. Obtain a spreadsheet token for your google sheets account
+    1. You must be added to the `carson-eng-testing` service account on the `partner-db-api` gcp project
```

Comment:
no this was for testing only. we must not expose those project names. 
you can mention, contact your IoT TPM to get granted access.

Reply:
Changed the wording here and added the TPM line.